src/components/Pools/RemoveFarm/index.tsx

```diff
@@ -181,25 +184,35 @@ const RemoveFarm = ({ stakingInfo, version, onClose, onLoading, onComplete, redi
                     <StatWrapper>
                       <Stat
                         title={t('earn.depositedToken', { symbol: 'PGL' })}
-                        stat={stakingInfo?.stakedAmount?.toSignificant(4)}
+                        stat={numeral(
+                          stakedAmount?.toFixed(
+                            stakedAmount?.greaterThan('0') && stakedAmount?.token?.decimals > 1 ? 2 : undefined,
+                          ),
+                        ).format('0.00a')}
```

Comment:
can you please add some comments here on why you did this change here ? this will help us in future to understand why we have done this.

Reply:
It fix this issue, I added in numeral to able to format this numer to 10k, 1M, 1B etc, to make that number smaller,
The function `.toFixed()` needs special care because as the decimals of pgl token in hedera is 0 we can't put `.tofixed(2)` because it can break the code and give a blank screen, for other chains where the decimals of pgl token  is greater than 1 we don't have this problem.
![image](https://user-images.githubusercontent.com/37405304/233638979-a156bd73-f7b2-4501-9527-ed95748672b5.png)
https://app.clickup.com/t/8669wgeva